src/CLICommand/action.test.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from 'chai';
+import { CLIAction } from './action';
+import { ParsedParameters } from './cli';
+import { ALL_VALID_EXIT_CODES, SUCCESS_EXIT_CODE } from './error_codes';
+
+describe('The CLIAction class', () => {
+	let action: CLIAction;
+	const params: ParsedParameters = {
+		myCustomOption: 'the parameter value'
+	};
+
+	it('The static CLIAction.runningAction throws when is read read before calling trigger()', () => {
```

Comment:
Its unclear to me what this means. I suggest rephrasing it

Reply:
What do you think about:
```suggestion
	it("There's no runningAction before calling trigger()", () => {
```

Reply:
Maybe something like this:

```suggestion
	it('throws an error when runningAction is called before `trigger` method', () => {
```

So the end result would read:

"The CLIAction class throws an error when runningAction is called before the `trigger` method"